jcl/src/java.base/share/classes/java/lang/J9VMInternals.java

```diff
@@ -503,6 +503,49 @@ static int fastIdentityHashCode(Object anObject) {
 	 */
 	static native int identityHashCode(Object anObject);
 
+	/*[IF INLINE-TYPES] */
+	/**
+	 * Answers an integer hash code for the parameter.
+	 * The caller must ensure that the parameter is a value type.
+	 * The hash code returned is the same one that would
+	 * be returned by java.lang.Object.hashCode(), assuming
+	 * the object's class has not overridden hashCode() and
+	 * that the parameter is a value type.
+	 *
+	 * @param		anObject	the object
+	 * @return		the hash code for the object
+	 * @throws		InternalError if the object is not a value type
+	 *
+	 * @see			java.lang.Object#hashCode
+	 */
+	static int valueHashCode(Object anObject) {
+		int hashcode = 0;
```

Comment:
Please remove the redundant initialization here (the Java compiler will issue an error if it can't prove it's always assigned before used).

Reply:
updated in 8533190